src/WebAppDIRAC/WebApp/handler/SystemAdministrationHandler.py

```diff
@@ -636,101 +633,30 @@ def web_getSelectionData(self):
     @asyncGen
     def web_ComponentLocation(self):
 
-        _setup = self.getUserSetup()
-        setup = _setup.split("-")[-1]
-
-        hosts = []
-        result = ComponentMonitoringClient().getHosts({}, False, False)
-        if result["OK"]:
-            hosts = [h["HostName"] for h in result["Value"] if h.get("HostName")]
-
-        componentTypes = ["Services", "Agents"]
+        componentTypes = ["Service", "Agent", "Executor"]
         componentTypes = self.get_arguments("ComponentType") or componentTypes
 
-        componentNames = []
-        if "ComponentName" in self.request.arguments:
-            componentNames = list(json.loads(self.get_argument("ComponentName")))
-
         componentModules = []
         if "ComponentModule" in self.request.arguments:
             componentModules = list(json.loads(self.get_argument("ComponentModule")))
 
-        showAll = 0
-        if "showAll" in self.request.arguments:
-            showAll = int(self.get_argument("showAll"))
-
```

Comment:
Do web portal no longer need option "showAll"?
The showAll option is also described in /WebAppDIRAC/WebApp/static/DIRAC/SystemAdministration/classes/SystemAdministration.js

Reply:
I removed it because `showAll` is unused in this module.